client/src/pages/RentalProcess.js

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const Form = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const AdressLine1 = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  div {
+    display: flex;
+    flex-direction: column;
+  }
+  & :first-child {
+    width: 70%;
+  }
+  & :last-child {
+    width: 20%;
+  }
+`;
+
+const AdressLine2 = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  div {
+    display: flex;
+    flex-direction: column;
+  }
+  & :first-child {
+    width: 35%;
+  }
+  & :last-child {
+    width: 55%;
+  }
+`;
+
+const DateSelect = styled.div`
+  display: flex;
+  justify-content: space-between;
+  & :first-child {
+    width: 45%;
+  }
+  & :last-child {
+    width: 45%;
+  }
+`;
+
+function RentalProcess() {
+  return (
+    <Form>
+      <label>
+        Anrede
+        <select required>
+          <option value="Herr">Herr</option>
+          <option value="Frau">Frau</option>
+        </select>
+      </label>
+      <label>
+        Vorname
+        <input placeholder="Vorname" required />
+      </label>
+      <label>
+        Nachname
+        <input placeholder="Nachname" />
+      </label>
+      <label>
+        E-Mail
+        <input placeholder="E-Mail" />
+      </label>
+      <label>
+        Telefonnummer
+        <input placeholder="Telefonnummer" />
+      </label>
+
+      <AdressLine1>
+        <div>
+          <label>Straße</label>
+          <input placeholder="Straße" />
```

Comment:
The label and input are not connected.
If you click on the label in your website, the input will not be focused.

That's why it's good to wrap the input inside the label :). 
Or you add a `id`, but make sure that it's unique.